### PR TITLE
prettier: max print witdh to 90

### PR DESCRIPTION
**Pull request by donnisnoni95**
It's better than the default one: 80.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -3,5 +3,6 @@
 	"singleQuote": true,
 	"tabWidth": 4,
 	"trailingComma": "none",
-	"useTabs": true
+	"useTabs": true,
+	"printWidth": 90
```
Comment (Alhadis) [inline .prettierrc]: `"printWidth": 90` should be listed at the top of the file, to keep it alphabetised.

Comment (donnisnoni95) [reply .prettierrc]: yeah... good point @Alhadis, BTW will push the formatted later. if merged

**Comment by donnisnoni95** [comment]
@Alhadis how can I find you on discord ??

**Comment by Alhadis** [comment]
I don't use Discord (or any messaging/chat applications, for that matter).

**Comment by donnisnoni95** [comment]
ok.. email??

**Comment by Alhadis** [comment]
It's on my profile, mate. 👍

**Comment by donnisnoni95** [comment]
Thank you... can you solve #53 ??

**Pull request merged**